sns/cm/dtrebalance/cm.c

```diff
@@ -223,6 +223,18 @@ m0_sns_cm_fid_dtrebalance_done(struct m0_fid *gfid, struct m0_reqh *reqh,
 	return SRS_UNINITIALIZED;
 }
 
+M0_INTERNAL int dtrebalance_cm_ag_next(struct m0_cm *cm,
```

Comment:
dtrebalance_cm_ag_next() could be dummy function

Reply:
Please elaborate.